sdk/python/lib/setup.py

```diff
@@ -44,7 +44,7 @@ def readme():
       # Keep this list in sync with Pipfile
       install_requires=[
           'protobuf~=4.21',
-          'grpcio~=1.59',
+          'grpcio==1.56.2',
```

Comment:
Could we bump to 1.58?

Reply:
s/58/57

Reply:
Possibly. I think I'd rather get us back on a known good version and take care of the upgrade after the upstream issue is fixed.